nih-sparc/sparc-app
# Wire Up Contentful to Project Details Page #408xt8

# Description

The purpose of this PR is to wire up the contentful data to the project details page. Please note the following:

- Datasets tab is not wired up yet due to pending completion of the endpoint
- Banner image is not yet in contentufl
- Responsive mobile design is not complete yet, which includes the following:
a) 150 character limit for description on mobile 
b) 90 character limit for title on mobile

## Type of change


# How Has This Been Tested?

1. Navigate to the projects tab on the Find Data page.
2. Click on a project title. The project details page for that project should load and display content as expected according to the designs.

# Checklist:
